### DISCUSSION TITLE
Only 1 decimal place in number

### DISCUSSION THREAD

**Discussion by GeorgeManikas** [strapi/strapi#7452] [Legacy Topics]
I have created a collection with a number field in it .. i declare it as a decimal number , but when i try to enter a record with more than 1 decimal place, it throws an error... For example , i set 0.95 as a value and it says that value must be between 0.9 and 1 ..  i have also tried to declare as a float number , same result... 
What am i doing wrong ? Thanks in advance

**Comment by MattieBelt** [comment]
Hey @GeorgeManikas this is a know issue: #7401
